modules/irmaseal-pkg-server.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.irmaseal-pkg;
+in
+{
+
+  options.services.irmaseal-pkg = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        This option enables the IRMAseal-PKG server.
+      '';
+    };
+
+    package = mkOption {
+      type = types.path;
+      default = pkgs.irmaseal-pkg;
+      description = "The IRMAseal-PKG package to use";
+    };
+
+    host = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = ''
+        Host to bind this service to
+      '';
+    };
+
+    irma = mkOption {
+      type = types.str;
+      default = "https://irma-noauth.demo.sarif.nl";
+      description = ''
+        URL of the IRMA go server to use for authentication
+      '';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 8087;
+      description = ''
+        TCP port to bind this service to
+      '';
+    };
+
+    publicKeyPath = mkOption {
+      type = types.path;
+      default = ./pkg.pub;
```

Comment:
Thinking some more about this, we can't provide a relative path as a default.
this would mean the user needs to have those keys inside the flake, but most likely they won't have access to it.
the default should be `/var/lib/irmaseal-pkg/pkg.pub` (and the corresponding sec version)